src/components/usa-banner/index.js

```diff
@@ -96,32 +121,6 @@ export class UsaBanner extends LitElement {
     return bannerActionText?.textContent;
   }
 
-  // TODO: Use inline image instead or translate strings.
```

Comment:
Re: "translate strings", would it still be reasonable to expect a text alternative for the image? I think an argument _could_ be made that the image is decorative since it's described by adjacent text ("a lock"), but I'm also not sure how a screen reader would handle effectively empty parentheses `()`.

Maybe either:

1. The `<span />` should be `role=img` with `aria-label`, as it was in https://github.com/uswds/uswds/pull/5829
2. Or the entire parenthesized content should be in a `aria-hidden` wrapper (`<span aria-hidden>(<span class="usa-banner__icon-lock"></span>)</span>`), unless testing confirms that assistive technology gracefully handles the parantheses

Reply:
@aduth good point. 

I'd consider it decorative, but will manually test in voiceover and find the closest approach to current Banner.

Reply:
Resolved in ca7978305e1274acccf91541841cc824e11ae9cb.